Example/JustTweak/Code/AppDelegate.swift

```diff
@@ -29,4 +30,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             window?.rootViewController?.present(alertController, animated: true, completion: nil)
         }
     }
+    
+    func makeTweakManager() -> TweakManager {
```

Comment:
nit: could make this  `var tweakManager: TweakManager `

Reply:
So it would have to be a lazy var instead of computed property or we would re-create the manager each time (I know, we currently call it only once). Is it really worth it?